experiments/gym/train_lunar_lander.py

```diff
@@ -163,25 +163,65 @@ def train_lunar_lander(args):
         ),
     ]
 
-    callbacks = logged_cbs + [
-        TensorboardLogger(
-            logged_cbs,
-            SummaryWriter(
-                log_dir=args.log_dir + "/" + args.name + "_{}".format(time.time())
+    if args.use_wandb:
+        import wandb
+
+        config = {
+            "wandb_project": args.name,
+            "wandb_run": args.wandb_run,
+            "hidden_dims": hidden_dims,
+            "batch_size": batch_size,
+            "learning_rate": learning_rate,
+            "rollout_len": rollout_len,
+        }
+
+        wandb_run = wandb.init(
+            project=config["wandb_project"],
+            name=config["wandb_run"],
+            config=wandb.helper.parse_config(
+                config, exclude=("wandb_project", "wandb_run")
```

Comment:
Why are we adding them to the config 10 lines up and then excluding them here? :D

What does "parse_config" do? It seems weird to give that a python-dict... sounds more like it would want a string?

Reply:
I was sending the full config to the logger first, when I switched it I must have forgotten to rewrite this part [🤦](https://emojipedia.org/person-facepalming/) 

Parse_config is used to exclude some keys from the dictionary so they are not logged as separate fields in the W&B table. It used to be a parameter passed to init(), but it is deprecated and they suggest using this instead.

Reply:
Nice! Do you know how it handles recursiveness? :-) I.e, if we have a dict inside the config dict.

Reply:
The way it works is the following: `config = {key: value for key, value in config.items() if key not in exclude}`, so in the case of a dict inside config, we could exclude the entire dict based on its key, but not a key inside of it.

Reply:
👍 Thank you! Sounds great.